src/test/javascript/unit/factories.spec.js

```diff
@@ -0,0 +1,19 @@
+describe('factories unit testing', function(){
+
```

Comment:
If that spec is used for testing other factories too then tests for restFactory can be combined into sub spec